test/index.js

```diff
@@ -240,14 +320,22 @@ describe('normalize.undefined', function() {
     expect(result).toEqual(value);
     done();
   });
+
+  it('rejects values that won\'t coerce to undefined', function(done) {
```

Comment:
I don't think so. GH obscured the separation here, but this test is in the `normalize.undefined` section.